yunwoo/programmers-1/크레인-인형뽑기-게임.js

```diff
@@ -0,0 +1,34 @@
+function solution(board, moves) {
+  const basket = [];
+  let result = 0;
+
+  moves.forEach((el) => {
+    for (let i = 0; i < board.length; i++) {
+      const stack = board[i][el - 1];
+
+      if (!stack) continue;
+
+      if (basket[basket.length - 1] !== stack) {
+        basket.push(stack);
+      } else {
+        basket.pop();
+        result += 2;
+      }
+
+      board[i][el - 1] = 0;
+      break;
+    }
+  });
+
+  return result;
+}
```

Comment:
```suggestion
function solution(board, moves) {
  const basket = [];
  let result = 0;

  moves.forEach((move) => {
    for (let i = 0; i < board.length; i++) {
      const stack = board[i][move - 1]; // stack!? 네이밍 고민!

      if (!stack) continue;

      if (basket[basket.length - 1] !== stack) {
        basket.push(stack);
      } else {
        basket.pop();
        result += 2;
      }

      board[i][move - 1] = 0;
      break;
    }
  });

  return result;
}
```
- 정석적인 풀이 잘봤습니다! 
- 위 풀이에서 stack의 의미가 뭘까요?? 자료 구조의 stack을 의미한다면 개별 요소를 할당하는 위 풀이에서의 stack은 적절한 네이밍이 아닌 것 같고, 다른 네이밍을 고민해보시는게 좋을 것 같습니다 !
- moves라는 변수를 forEach로 순회하기 때문에 각 개별 요소는 el이 아닌 move가 더욱 적절해보입니다!

Reply:
- 아아 뽑은 인형의 배열이 stack이 되었어야 하는군요..!
- move ! 변수명 조금 더 신경 써보겠습니다 코멘트 감사합니다 🙇‍♂️!!!